balloon/src/main/java/com/skydoves/balloon/Balloon.kt

```diff
@@ -240,6 +243,11 @@ class Balloon(
     }
   }
 
+  private fun prepareLayoutDirection() {
+    horizontalLayoutFactor = if (builder.isRtl) -1
+    else 1
```

Comment:
The default value is already `1`.
How about this?
```kotlin
if (builder.isRtl) {
  horizontalLayoutFactor = -1
}
```

Reply:
done